src/types/components/accordions/accordions.interface.ts

```diff
@@ -0,0 +1,13 @@
+import { SxProps } from "@mui/material"
+
+export interface AccordionItem {
+    title: string
+    description: string
+}
+interface NestedStyles {

```

Comment:
don't use this , withIcon and noIcon have certain keys you can create interface for them

Reply:
certain keys?for example?

Reply:
certain interface is unnecessary, because all fields are partial

Reply:
every field has no required for using in styles

Reply:
what i see in your code is accordion,summary,titleActive,etc

Reply:
all of them are optional for adding in styles so why need to write interface for this styles?

Reply:
they will be needed to omit as you have written about noIcon

Reply:
because other fields which will can be provided to [key:string]:bla bla, will have no effect on your component, so it has to be typed